msm/src/serialization/interpreter.rs

```diff
@@ -802,10 +802,9 @@ mod tests {
             lookups::LookupTable,
             N_INTERMEDIATE_LIMBS,
         },
-        Ff1, LIMB_BITSIZE, N_LIMBS,
+        Ff1, Fp, LIMB_BITSIZE, N_LIMBS,
```

Comment:
Not that this is not the same field. `Fp` points to `ark_bn254::Fp` in `msm/src/lib.rs`. Is it expected to change the field?

Reply:
OK indeed you're right. I have no idea why this module uses `native = foreign` in tests instead, but I returned it back/made it clear now.

Reply:
quite confusing actually, I spent some time thinking about the same issue in the other file :)